src/components/activitylog.js

```diff
@@ -84,16 +77,9 @@ function reloadData(instance, elem, apiClient, startIndex, limit) {
     })).then(function (result) {
         elem.setAttribute('data-activitystartindex', startIndex);
         elem.setAttribute('data-activitylimit', limit);
-        if (!startIndex) {
+        if (startIndex === 0) {
             const activityContainer = dom.parentWithClass(elem, 'activityContainer');
```

Comment:
it looks like this assignment can be removed now and just combine the element selection with the class toggle statement